lib/node_modules/@stdlib/stats/base/varianceyc/lib/ndarray.js

```diff
@@ -47,31 +53,38 @@
 * var v = varianceyc( N, 1, x, 2, 1 );
 * // returns 6.25
 */
-function varianceyc( N, correction, x, stride, offset ) {
+function varianceyc( N, correction, x, strideX, offsetX ) {
 	var sum;
 	var ix;
 	var S;
 	var v;
 	var d;
 	var n;
 	var i;
+	var o;
+
+	o = arraylike2object( x );
+	if ( o.accessorProtocol ) {
+		return accessors( N, correction, o, strideX, offsetX );
+	}
 
```

Comment:
This logic should be moved to L78 so that we don't need to repeat it in `accessors.js`.